content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,139 @@
+Non-global variables are only applied to a subset of workspaces in an organization.
 
-### Looking Up Variable Names
+When non-global variable sets have conflicting variables, Terraform Cloud uses values from the variable set that was applied most recently. For example, if you apply Variable Set A and then apply Variable Set B to the same workspace a week later, Terraform Cloud will use any conflicting variables from Variable Set B. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the date that each variable set is applied to the workspace when determining precedence.
 
-Terraform Cloud can't automatically discover variable names from a workspace's Terraform code. You must discover the necessary variable names by reading code or documentation, then enter them manually.
+### 3. Global Variable Sets
 
-If a required input variable is missing, Terraform plans in the workspace will fail and print an explanation in the log.
+Non-global variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. This is true regardless of when the global variable set was applied.
 
-### Descriptions
+### 4. Variable Files
 
-Variable descriptions help distinguish between similarly named variables. These optional fields are only shown on this "Variables" page and are completely independent from any variable descriptions declared in Terraform CLI.
+When variables from [committed `.auto.tfvars` files](#loading-variables-from-files) have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
 
-## How Terraform Cloud Uses Variables
 
-### Terraform Variables
 
-Terraform Cloud passes variables to Terraform by writing a `terraform.tfvars` file and passing the `-var-file=terraform.tfvars` option to the Terraform command.
+### Precedence Example
 
-Do not commit a file named `terraform.tfvars` to version control, since Terraform Cloud will overwrite it. (Note that you shouldn't check in `terraform.tfvars` even when running Terraform solely on the command line.)
+Consider an example workspace that has:
 
-### Environment Variables
+| Workspace-Specific Variables | Variable Set A | Variable Set B (Global)|
+| -----------------------------|----------------|----------------|
+|   | Applied 10/4 | Applied 10/20 |
+| `ACCESS_KEY` = `g`, `ACCESS_ID` = `q`, `VAR1` = `h` | `KEY1` = `x`, `VAR1` = `y` | `KEY1` = `z`, `VAR2` = `a` |
```

Comment:
Can we make the keys the row titles in the table, and give each variable its own row? That would make it easier to tell which variables are overlapping between sources.

Reply:
yes! good idea - I did that just now and it's much better!